base/src/main/java/com/smartdevicelink/managers/screen/TextAndGraphicUpdateOperation.java

```diff
@@ -555,8 +621,8 @@ private boolean sdlArtworkNeedsUpload(SdlArtwork artwork) {
      * @return true if primaryGraphic should be updated, false if not
      */
     private boolean shouldUpdatePrimaryImage() {
-        boolean templateSupportsPrimaryArtwork = templateSupportsImageField(ImageFieldName.graphic);
-        String currentScreenDataPrimaryGraphicName = (currentScreenData != null && currentScreenData.getGraphic() != null) ? currentScreenData.getGraphic().getValue() : null;
+        boolean templateSupportsPrimaryArtwork = templateSupportsImageField(ImageFieldName.graphic) || shouldUpdateTemplateConfig();
```

Comment:
We are just sending everything if shouldUpdateTemplateConfig is true because we don't know the capabilities of the new template at this time, when the capabilities change in the TextAndGraphicsManager it will send another update and correct anything that could be wrong